Watch Extension/MRSessionProgressGroupRenderer.swift

```diff
@@ -11,12 +11,18 @@ class MRSessionProgressGroupRenderer : NSObject {
         self.timer = NSTimer.scheduledTimerWithTimeInterval(1, target: self, selector: "update", userInfo: nil, repeats: true)
     }
     
-    deinit {
+    func deactivate() {
         self.timer?.invalidate()
         self.timer = nil
     }
     
+    deinit {
+        deactivate()
+    }
+    
     func update() {
+        MRExtensionDelegate.sharedDelegate().applicationDidBecomeActive()
```

Comment:
It makes it easier to get the samples: one doesn’t have to swipe up and down like mad to activate the glance to try to get the date over to the phone.